ulauncher/modes/extensions/extension_mode.py

```diff
@@ -94,7 +95,10 @@ def activate_result(self, result: Result, query: Query, alt: bool) -> ActionMeta
         Called when a result is activated.
         Override this method to handle the activation of a result.
         """
-        handler = getattr(result, "on_alt_enter" if alt else "on_enter", DEFAULT_ACTION)
+        handler = cast(
+            "ActionMetadata | Callable[[Query], ActionMetadata]",
+            getattr(result, "on_alt_enter" if alt else "on_enter", DEFAULT_ACTION),
+        )
```

Comment:
We didn't have types for this. Mypy would just infer this as any, while pyrefly inferred as `bool | Any`